spk/sabnzbd/src/wizard/upgrade_uifile

```diff
@@ -1,9 +1,64 @@
-[{
+[
```

Comment:
So wizards are supported? I am so confused 😨

Reply:
As far as I understand it. For now they work as before but officially they are no longer supported by Synology. I hope Synology can come up with an alternative. Their own packages and 3rd party still use them. So I think eventually we have to phase them out by setting default values.

Reply:
👍